### PR TITLE
Add option to disable linking to missing tiddlers

### PR DESCRIPTION
**Pull request by sukima**
This closes Issue #2340

### CODE REVIEW

File: core/modules/widgets/link.js [hunk 1]
```diff
@@ -225,7 +229,8 @@ Selectively refreshes the widget if needed. Returns true if the widget or any of
 */
 LinkWidget.prototype.refresh = function(changedTiddlers) {
 	var changedAttributes = this.computeAttributes();
-	if(changedAttributes.to || changedTiddlers[this.to] || changedAttributes["aria-label"] || changedAttributes.tooltip) {
+	console.log('config changed:', changedTiddlers[MISSING_LINK_CONFIG_TITLE]);
```
Comment (pmario) [inline core/modules/widgets/link.js]: imo should be disabled for PR.

File: core/modules/widgets/link.js [hunk 2]
```diff
@@ -35,9 +36,10 @@ LinkWidget.prototype.render = function(parent,nextSibling) {
 	this.execute();
 	// Get the value of the tv-wikilinks configuration macro
 	var wikiLinksMacro = this.getVariable("tv-wikilinks"),
-		useWikiLinks = wikiLinksMacro ? (wikiLinksMacro.trim() !== "no") : true;
+		useWikiLinks = wikiLinksMacro ? (wikiLinksMacro.trim() !== "no") : true,
+		missingLinksEnabbled = !(this.hideMissingLinks && this.isMissing);
```
Comment (pmario) [inline core/modules/widgets/link.js]: missingLinksEnabbled ... typo?

File: core/modules/widgets/link.js (hunk 1 above)
Comment (sukima) [inline core/modules/widgets/link.js]: Good catch.

File: core/modules/widgets/link.js (hunk 2 above)
Comment (sukima) [inline core/modules/widgets/link.js]: Totally a typo… rebasing

**Comment by Jermolene** [comment]
Thanks @sukima sorry for the delay, but this looks good to me.

**Pull request merged**